Makefile

```diff
@@ -61,7 +61,18 @@ $(VALIDATION_TESTS): %.t: %.go
 print-validation-tests:
 	@echo $(VALIDATION_TESTS)
 
-.PHONY: test .gofmt .govet .golint print-validation-tests
+.PHONY: test .gofmt .govet .golint print-validation-tests vendor validate-vendor
+
+vendor:
+	GO111MODULE=on go mod tidy
+	GO111MODULE=on go mod vendor
```

Comment:
do we need to turn this on explicitly ?

Reply:
So, I added it,  because go 1.11 won't use go modules if we're anywhere inside GOPATH. That said; contributors very likely won't be on go 1.11, and I now exclude this in CI, so I can remove it now, or in the same PR where I will remove Go 1.11 from the matrix (I think go 1.11 could probably be removed by now, unless we want to continue verifying that older go versions still build)